io/src/pcd_grabber.cpp

```diff
@@ -387,7 +387,6 @@ pcl::PCDGrabberBase::PCDGrabberBase (const std::vector<std::string>& pcd_files,
 ///////////////////////////////////////////////////////////////////////////////////////////
 pcl::PCDGrabberBase::~PCDGrabberBase () throw ()
 {
-  stop ();
   delete impl_;
```

Comment:
Okay. It's not clear why they were made public in the first place. Anyways, that's a separate issue. Could you put the opening brace in dtor on a separate line so I can merge?